src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -85,6 +85,22 @@ const safeAppWeb3Provider = new Web3.providers.HttpProvider(getSafeAppsRpcServic
 const URL_NOT_PROVIDED_ERROR = 'App url No provided or it is invalid.'
 const APP_LOAD_ERROR = 'There was an error loading the Safe App. There might be a problem with the App provider.'
 
+// Some apps still need chain name, as they didn't update to chainId based SDK versions
+// With naming changing in the config service some names aren't the expected ones
+// Ex: Ethereum -> MAINNET, Gnosis Chain -> XDAI
+const getLegacyChainName = (chainName: string) => {
```

Comment:
Would it make sense to rather map the chainId, as we should not assume that the name will not change

Reply:
This way we only map the name of the two networks that changed. Is not optimal because it changed once, and could change again, but using the chainId will make mandatory for us to keep this code up to date for all chains and whenever a new chain is added to the config service.

Reply:
Why would you've had to keep it up to date for all chains? I think this suggestion makes sense

Reply:
Added a new proposal based on this suggestion